src/connector/src/sink/encoder/mod.rs

```diff
@@ -150,6 +150,33 @@ pub enum CustomJsonType {
     None,
 }
 
+/// How the jsonb type is encoded.
+///
+/// - `String`: encode jsonb as string. `[1, true, "foo"] -> "[1, true, \"foo\"]"`
+/// - `Dynamic`: encode jsonb as json type dynamically. `[1, true, "foo"] -> [1, true, "foo"]`
+/// - `Custom`: decided by [`CustomJsonType`].
+pub enum EncodeJsonbMode {
+    String,
+    Dynamic,
+    Custom,
+}
+
+impl EncodeJsonbMode {
+    pub const OPTION_KEY: &'static str = "encode.jsonb.mode";
```

Comment:
Any reason not following the existing naming pattern `jsonb.handling.mode`?

Reply:
You are right. I should follow the naming convention.